warehouse/query-core/src/main/java/datawave/query/iterator/logic/TermFrequencyExcerptIterator.java

```diff
@@ -302,11 +346,13 @@ public void next() throws IOException {
             return;
         }
 
+        final int startOffset = config.startOffset;
+        final int endOffset = config.endOffset;
+        final List<String> hitTermsList = config.hitTermsList;
+        final String fieldName = config.fieldName;
+
         // get the pieces from the top key that will be returned
         Key top = source.getTopKey();
-        Text cv = top.getColumnVisibility();
-        long ts = top.getTimestamp();
-        Text row = top.getRow();
```

Comment:
This as a little dangerous because `top` can change between here and where these values were used. We can just call the methods on `Key` to get these values when we need them.